changelogs/CHANGELOG-v1.22.0.md

```diff
@@ -32,6 +29,13 @@ For information on getting started with Contour and Gateway API, see the [Contou
 
 # Minor Changes
 
+## Update to Envoy 1.23.0
```

Comment:
Note the contents of the changelog should be the same as for the RC, I just reordered a few things to be more logical.